code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -433,9 +424,7 @@
 	helptext = "We must constantly repair our form to make it space-proof, reducing chemical production while we are protected. Retreating the suit damages our genomes. Cannot be used in lesser form."
```

Comment:
Remove the thing saying it damages your genomes
There's also the same thing for the armour